### PR TITLE
Edited info cmd text

### PR DESCRIPTION
**Pull request by tech6hutch**
### Description of the PR
See #79 (finally getting around to this)

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

- Change the info command text (consistent with Komada's changes)

### Semver Classification
I guess this counts as a non-code change ¯\\\_(ツ)\_/¯
- [x] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [ ] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

**Comment by Tylertron1998** [comment]
In honesty, I think its better without the Emoji's; it looks more professional. Other then that, good job, I approve!

**Comment by tech6hutch** [comment]
https://github.com/dirigeants/komada/blob/master/src/commands/System/info.js Komada has them ¯\\\_(ツ)\_/¯

File: src/languages/en-US.js [hunk 1]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
+				'• 💬 Customizable command system with automated usage parsing and the ability to load/reload commands on-the-fly',
+				'• 👀 "Monitors", which can watch messages and act on them, like a normal message event (for swear filters, spam protection, etc.)',
+				'• ⛔ "Inhibitors", which can prevent commands from running based on a set of parameters (for permissions, blacklists, etc.)',
+				'• 🗄 "Providers", which standardize and simplify usage of outside databases of your choosing',
+				'• ✅ "Finalizers", which run on messages after a successful command',
+				'• ➕ "Extendables", which passively add properties or methods to existing Discord.js or Klasa classes',
+				'• 🌐 "Languages", which allow you to localize your bot',
 				'',
```
Comment (bdistin) [inline src/languages/en-US.js]: missing "Tasks"

File: src/languages/en-US.js [hunk 2]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
-				'• "Extendables", code that acts passively. They add properties or methods to existing Discord.js classes.',
-				'• "Languages", which allow you to localize your bot.',
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
+				'• 💬 Customizable command system with automated usage parsing and the ability to load/reload commands on-the-fly',
+				'• 👀 "Monitors", which can watch messages and act on them, like a normal message event (for swear filters, spam protection, etc.)',
+				'• ⛔ "Inhibitors", which can prevent commands from running based on a set of parameters (for permissions, blacklists, etc.)',
+				'• 🗄 "Providers", which standardize and simplify usage of outside databases of your choosing',
+				'• ✅ "Finalizers", which run on messages after a successful command',
+				'• ➕ "Extendables", which passively add properties or methods to existing Discord.js or Klasa classes',
```
Comment (bdistin) [inline src/languages/en-US.js]: `add static properties or methods of any kind to`...

**Comment by Tylertron1998** [comment]
I know, I prefer it this way; without them.

File: src/languages/en-US.js [hunk 3]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
-				'• "Finalizers" which run on messages after a successful command.',
-				'• "Extendables", code that acts passively. They add properties or methods to existing Discord.js classes.',
-				'• "Languages", which allow you to localize your bot.',
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
+				'• 💬 Customizable command system with automated usage parsing and the ability to load/reload commands on-the-fly',
+				'• 👀 "Monitors", which can watch messages and act on them, like a normal message event (for swear filters, spam protection, etc.)',
+				'• ⛔ "Inhibitors", which can prevent commands from running based on a set of parameters (for permissions, blacklists, etc.)',
+				'• 🗄 "Providers", which standardize and simplify usage of outside databases of your choosing',
+				'• ✅ "Finalizers", which run on messages after a successful command',
```
Comment (bdistin) [inline src/languages/en-US.js]: `which run after a successful commands to allow collection of stats, logging, cleanup of responses, ect`

File: src/languages/en-US.js [hunk 4]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
-				'• "Providers" which allow you to connect with an outside database of your choosing.',
-				'• "Finalizers" which run on messages after a successful command.',
-				'• "Extendables", code that acts passively. They add properties or methods to existing Discord.js classes.',
-				'• "Languages", which allow you to localize your bot.',
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
+				'• 💬 Customizable command system with automated usage parsing and the ability to load/reload commands on-the-fly',
+				'• 👀 "Monitors", which can watch messages and act on them, like a normal message event (for swear filters, spam protection, etc.)',
+				'• ⛔ "Inhibitors", which can prevent commands from running based on a set of parameters (for permissions, blacklists, etc.)',
+				'• 🗄 "Providers", which standardize and simplify usage of outside databases of your choosing',
```
Comment (bdistin) [inline src/languages/en-US.js]: simplify usage of __any__ databases of your choosing

File: src/languages/en-US.js [hunk 5]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
+				'• ✅ "Finalizers", which run on messages after a successful command',
+				'• ➕ "Extendables", which passively add properties or methods to existing Discord.js or Klasa classes',
+				'• 🌐 "Languages", which allow you to localize your bot',
```
Comment (bdistin) [inline src/languages/en-US.js]: `your bot's responses`

File: src/languages/en-US.js [hunk 6]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
-				'• "Inhibitors" which can prevent commands from running based on a set of parameters (Permissions, Blacklists, etc)',
-				'• "Providers" which allow you to connect with an outside database of your choosing.',
-				'• "Finalizers" which run on messages after a successful command.',
-				'• "Extendables", code that acts passively. They add properties or methods to existing Discord.js classes.',
-				'• "Languages", which allow you to localize your bot.',
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
+				'• 💬 Customizable command system with automated usage parsing and the ability to load/reload commands on-the-fly',
+				'• 👀 "Monitors", which can watch messages and act on them, like a normal message event (for swear filters, spam protection, etc.)',
+				'• ⛔ "Inhibitors", which can prevent commands from running based on a set of parameters (for permissions, blacklists, etc.)',
```
Comment (bdistin) [inline src/languages/en-US.js]: `which can prevent commands from running based on any condition you wish to apply`

File: src/languages/en-US.js [hunk 7]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
-				'• Customizable Command system with automated usage parsing and easy to use reloading and downloading modules',
-				'• "Monitors" which can watch messages and act on them, like a normal message event (Swear Filters, Spam Protection, etc)',
-				'• "Inhibitors" which can prevent commands from running based on a set of parameters (Permissions, Blacklists, etc)',
-				'• "Providers" which allow you to connect with an outside database of your choosing.',
-				'• "Finalizers" which run on messages after a successful command.',
-				'• "Extendables", code that acts passively. They add properties or methods to existing Discord.js classes.',
-				'• "Languages", which allow you to localize your bot.',
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
+				'• 💬 Customizable command system with automated usage parsing and the ability to load/reload commands on-the-fly',
```
Comment (bdistin) [inline src/languages/en-US.js]: ~~usage parsing~~ parameter resolving

File: src/languages/en-US.js [hunk 8]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
-				'• Fast Loading times with ES7 Support (Async/Await)',
-				'• Per-server configuration, that can be extended with your own code',
-				'• Customizable Command system with automated usage parsing and easy to use reloading and downloading modules',
-				'• "Monitors" which can watch messages and act on them, like a normal message event (Swear Filters, Spam Protection, etc)',
-				'• "Inhibitors" which can prevent commands from running based on a set of parameters (Permissions, Blacklists, etc)',
-				'• "Providers" which allow you to connect with an outside database of your choosing.',
-				'• "Finalizers" which run on messages after a successful command.',
-				'• "Extendables", code that acts passively. They add properties or methods to existing Discord.js classes.',
-				'• "Languages", which allow you to localize your bot.',
+				'• 🐇💨 Fast loading times with ES8 support (`async`/`await`)',
```
[the remark this answers is not in the document]
Comment (Tylertron1998) [reply src/languages/en-US.js]: plz, no emojis

File: src/languages/en-US.js [hunk 9]
```diff
@@ -110,15 +110,15 @@ module.exports = class extends Language {
...
+				'• 🎚🎛 Per-server settings that can be extended with your own fields',
```
[the remark this answers is not in the document]
Comment (tech6hutch) [reply src/languages/en-US.js]: Ah, true

File: src/languages/en-US.js (hunk 1 above)
Comment (tech6hutch) [reply src/languages/en-US.js]: Ah, forgot they're a thing now, lol

File: src/languages/en-US.js (hunk 2 above)
Comment (tech6hutch) [reply src/languages/en-US.js]: If it's going to say that, shouldn't it say both normal and static? `which passively add instance or static properties or methods`?

Comment (bdistin) [reply src/languages/en-US.js]: instance properties aren't possible with extendables.

Comment (tech6hutch) [reply src/languages/en-US.js]: But both instance and static methods are possible, right?

Comment (tech6hutch) [reply src/languages/en-US.js]: Wait, isn't this an instance property? https://github.com/dirigeants/klasa/blob/master/src/extendables/postable.js It just happens to be a getter

Comment (bdistin) [reply src/languages/en-US.js]: methods of any kind, being: ['static', 'instance', 'getter', 'setter']

Comment (kyranet) [reply src/languages/en-US.js]: What about this?
```javascript
const { Extendable } = require('klasa');

class MyExtendable extends Extendable {

    constructor(...args) {
        super(...args, {
            appliesTo: [],
            name: 'nameOfExtendable',
            enabled: true,
            klasa: false
        });
    }

}

MyExtendable.prototype.extend = 'wew'; // Make a instance property (field)

module.exports = MyExtendable;
```

Comment (bdistin) [reply src/languages/en-US.js]: That both doesn't work, and shouldn't be done.

Comment (tech6hutch) [reply src/languages/en-US.js]: I see. I tend to think of getters and setters more as "magic" properties than as methods.

**Comment by tech6hutch** [comment]
Made requested edits, and clarified a little

**Comment by gc-au** [comment]
Here is how I would do it. It should be concise, simple and clean IMO. The longer and more complex it is, the less likely it is to be fully read and grab attention of the reader. Make it understandable for noobs, but still let experienced bot developers understand the potential. I think it would also be highly beneficial to put a link at the end with an example of a command, e.g. a ping/pong command (pastebin link or something)
=====================
Komada is a Discord Bot framework that uses Discord.JS, that allows you to go straight to making your commands. It doesn't limit you at all, and will make it faster and easier for you to develop new features. 

• **Commands:** Make any kind of command in a simple, consistent format.
• **Monitors:** Do something anytime a message is posted anywhere.
• **Inhibitors:** Stop certain people from running certain commands.
• **SettingGateway:** Easily store data for guilds and users.
• **Finalizers:** Do something whenever a command is ran.
• **Functions:** Store functions in the client, use them *anywhere*.

**Comment by tech6hutch** [comment]
I agree with the notion of simplification.

**Comment by gc-au** [comment]
Klasa version. P.S if embeds were allowed you could make the feature names clickable, e.g. clicking on "Commands" in the list would take you to the commands page on klasa.js.org, also a blue color on the left, and neatly fit klasas logo in the thumbnail, author or footer fields. Although I can definitely understand not wanting to use embeds.


Klasa is a class-based discord bot framework built ontop of Discord.JS. It allows you to go straight to making your commands. It doesn't limit you at all, and will make it faster and easier for you to develop new features. Klasa is updated daily with new features, bug fixes, and performance improvements. 

Some features of Klasa include:

• **Commands:** Make commands in a simple, consistent format.
• **Klasa-Pieces:** A repository of free-to-use commands and features.
• **SettingGateway:** Easily store data for guilds and users.
• **Monitors:** Do something anytime a message is sent anywhere.
• **Inhibitors:** Stop certain people from running certain commands.
• **Finalizers:** Do something whenever a command runs.
• **Providers:** Easily store data in a database of your choice.
• **Languages:** Easily translate your bot into several languages.
• **Extendables:** Add properties/methods to existing Discord.js classes.

<https://klasa.js.org/>

**Pull request merged**